modules/src/application/ics20_fungible_token_transfer/relay_application_logic/send_transfer.rs

```diff
@@ -15,9 +15,7 @@ where
 {
     let source_channel_end = ctx
         .channel_end(&(msg.source_port.clone(), msg.source_channel.clone()))
-        .ok_or_else(|| {
-            Error::channel_not_found(msg.source_port.clone(), msg.source_channel.clone())
-        })?;
+        .map_err(Error::ics04_channel)?;
```

Comment:
Since the errors being returned have changed now, it would be nice to add missing context, i.e. `port_id` and `channel_id` to `MissingNextRecvSeq`, `MissingNextSendSeq` and `MissingNextAckSeq`. 
Also, `Ics20Error::SequenceSendNotFound` and `ChannelNotFound` can be safely deleted.